src/controllers/zendeskController.ts

```diff
@@ -92,6 +95,18 @@ export default class ZendeskController {
             jwt: token,
           })
         ),
+        TE.mapLeft((err) => {
```

Comment:
I don't know if it works, but we can use a `orElseFirst` instead of a map that doesn't execute a map.
```typescript
TE.orElseFirst((err) =>
  TE.of(
    this.appInsightsTelemetryClient?.trackEvent({
      name: "appbackend.zendesk.error",
      properties: {
        fiscal_code: sha256(user.fiscal_code),
        message: err.detail,
      },
      tagOverrides: { samplingEnabled: "false" },
    })
  )
),
```

Reply:
I'll try it for the next one. Thanks :)